src/components/Annotations/Shape.tsx

```diff
@@ -57,7 +79,7 @@ export function Shape(props: AnnotationShapeProps) {
 
   return (
     <g transform={`translate(${x}, ${y})`}>
-      <Figure size={props.size} style={props.style} />
+      <Figure size={props.size} style={props.style} color={props.color} />
```

Comment:
you could do `style={{ fill: props.color, ...props.style }}` here and remove the changes from the shapes above.

Reply:
fixed